Makefile

```diff
@@ -6,7 +6,9 @@ endif
 build: certs releases images
 
 certs:
+	[ -n ${DOMAIN} ] && sed -i "s/^DOMAIN=.*/DOMAIN=${DOMAIN}/g" ${GIT_ROOT}/env/defaults.env
```

Comment:
""-quote DOMAIN.
Further, add `-e` to indicate the sed-script.
Also maybe move the `-i` to in front of the path, like in the other `sed`-invokations I saw.